libs/external-db-mysql/src/mysql_capabilities.ts

```diff
@@ -0,0 +1,18 @@
+import {
+    CollectionOperation,
+    DataOperation,
+    FieldType,
+} from '@wix-velo/velo-external-db-types'
+
+const {
+    query,
+    count,
+    queryReferenced,
+    aggregate,
+    truncate
+} = DataOperation
+
+export const ReadWriteOperations = Object.values(DataOperation)
+export const ReadOnlyOperations = [query, count, queryReferenced, aggregate, truncate]
```

Comment:
@michaelir we need your help here, from a practical point of view, there is nothing that limits us to truncate a table when there is no an `_id` column (that's what read-only mean).
But from a definition point of view, it may indeed be strange that you can truncate a table in read-only mode. 
what do you think, should we include this data operation in the read only data operations array?